packages/contracts/scripts/generate-markdown.js

```diff
@@ -74,7 +82,7 @@ const networks = {
       if (contracts[i] === 'OVM_L1CrossDomainMessenger') {
         proxiedContracts.push(contracts.splice(i, 1)[0])
       }
-      if (contracts[i] === 'OVM_L1ETHGateway') {
+      if (contracts[i] === 'PROXY__OVM_L1ETHGateway') {
```

Comment:
What is this change for?

Reply:
@elenadimitrova  said she renamed the contract.

Reply:
Oh I think the correct diff here is `OVM_L1ETHGateway` => `OVM_L1StandardBridge`